stake-pool/js/src/index.ts

```diff
@@ -0,0 +1,172 @@
+import * as schema from './schema.js';
+import solanaWeb3 from '@solana/web3.js';
+
+/**
+ * Sample code to demonstrate how to use the JS bindings
+ * Also contains several useful helper functions
+ */
+
+export class StakePool {
+  /**
+   * Wrapper class for a stake pool.
+   * Each stake pool has a stake pool account and a validator list account.
+   */
+  stakePool: decodedStakePool;
+  validatorList: decodedValidatorList;
+}
+
+export interface decodedStakePool {
```

Comment:
I think the type names can be reversed:
```suggestion
export interface StakePoolAccount {
```